aclk/aclk_rrdhost_state.h

```diff
@@ -30,6 +30,7 @@ typedef enum aclk_agent_state {
 
 typedef struct aclk_rrdhost_state {
     char *claimed_id; // Claimed ID if host has one otherwise NULL
+    char *prev_claimed_id; // Claimed ID if changed (reclaimed) during runtime
```

Comment:
clear it on shutdown?
perhaps in `rrdhost_free` ?

Reply:
Correct! Pushed, it but leaving the converstation un-resolved to check.